model/lifecycle.go

```diff
@@ -681,7 +681,10 @@ func createTasksForBuild(project *Project, buildVariant *BuildVariant, b *build.
 			}
 			execTaskIds = append(execTaskIds, execTaskId)
 		}
-		t := createDisplayTask(id, dt.Name, execTaskIds, buildVariant, b, v, project)
+		t, err := createDisplayTask(id, dt.Name, execTaskIds, buildVariant, b, v, project)
+		if err != nil {
+			return tasks, err
```

Comment:
probably would be a good idea to wrap/annotate this and the reset of the errors added in this file

Reply:
Good point, thanks. Done.